localstack/services/sns/sns_listener.py

```diff
@@ -696,7 +696,7 @@ def publish_batch(topic_arn, req_data, headers):
         data = {}
         data["TopicArn"] = [topic_arn]
         data["Message"] = [message["Message"]]
-        data["Subject"] = [message["Subject"]]
+        data["Subject"] = [message.get("Subject", None)]
```

Comment:
There is no need to explicitly return None on get, therefore I would just remove it.